Models/DiaryFood.model.ts

```diff
@@ -0,0 +1,29 @@
+import {FoodDetails} from "./FoodDetails.model";
+
+type MealMap = {
+    [key: string]: FoodDetails[];
```

Comment:
I think that using [key in Meals]?: FoodDetails[]; instead of this line would be better.

Reply:
Hey, that's pretty good suggestion but the point is that if we developed more application users could add their own meals - ofc. I can change it and limit just to fixed meals, should I?

Reply:
I suggest to talk about it with whole team on meeting, because it depends on app requirements.
OK. I'll merge this branch.